.githooks/pre-commit

```diff
@@ -83,4 +83,4 @@ if [ $common_changed = true ]; then
     echo "ERROR: common/src/index.js was changed, but a yarn.lock file was not updated. Please make sure you've updated both yarn.lock files if you've updated the common/index.js file."
     exit 1
   fi
-fi
+fi
```

Comment:
What is this?

Reply:
looks like some git weirdness, safe to ignore